wicamo96/Verti-Comps
# Validate

## What Changed

- Added validation features
- Added a page to sort through competitions and competitors to view and validate competitor's climbs
- Added a page to include notes on why a competitor's climb was flagged

## How To Test

- Log in as administrator
- Click the navigate link in the navbar
- Select the competition you'd like to judge from the competitions dropdown
- Select which competitor you'd like to judge from the competitors dropdown
- Mark each climb as validated or flagged
- If flagged, click the add note link
- Add a note in the input on the next page
- Return to the validation page to see your changes
